kedro-telemetry/kedro_telemetry/masking.py

```diff
@@ -81,16 +81,19 @@ def _get_cli_structure(
     return output
 
 
-def _mask_kedro_cli(
-    cli_struct: dict[str | None, Any], command_args: list[str]
-) -> list[str]:
+def _mask_kedro_cli(cli: click.CommandCollection, command_args: list[str]) -> list[str]:
     """Takes a dynamic vocabulary (based on `KedroCLI`) and returns
     a masked CLI input"""
     output = []
-
-    # Preserve the initial part of the command until parameters sections begin
     arg_index = 0
-    current_CLI = cli_struct.get("kedro", {})
+    cmd = command_args[0] if command_args else ""
+    if cmd in {"--help", "--version", "-h", "-v", ""}:
+        return command_args
```

Comment:
Perhaps we only need to return `cmd` instead of the entire `command_args`, to avoid capturing any unnecessary or irrelevant input that might follow.

Reply:
If we return `cmd` here, the masked command would be `kedro ****` even when it's `kedro --help` etc. This sort of masking also takes place when it's a wrong command `kedro typo` is masked as `kedro ****`. Maybe it's worth having that distinction? WDYT?

Reply:
I agree, it makes sense